config/config.go

```diff
@@ -66,6 +66,34 @@ func NewLocal(cfgPath, basepath string) (*Local, error) {
 	return cfg, nil
 }
 
+// Validate validates the values of given configuration
+func (l *Local) Validate() []error {
+	var errors []error
+
+	if err := l.validateIssueTracker(); err != nil {
+		errors = append(errors, err)
+	}
+
+	return errors
+}
+
+func (l *Local) validateIssueTracker() error {
+	validIssueTrackers := []IssueTracker{
+		IssueTrackerJira,
+		IssueTrackerGithub,
+		IssueTrackerGitlab,
+		IssueTrackerPivotal,
+		IssueTrackerRedmine,
+	}
```

Comment:
Do you think it would be better to put this block somewhere close to the enum declaration?

Unfortunately, there is no way to foreach an enum in go, without this supplementary array, but we could at least make our lives easier by making it so when someone adds an enum in the future, the additional changes that have to be made is closer & easier to spot.

Reply:
Totally agreed! I'll change it.

Reply:
Done.